CRM/Contact/Page/AJAX.php

```diff
@@ -745,35 +745,35 @@ public static function getDedupes() {
     if (!empty($columnDetails)) {
       switch ($columnDetails['data']) {
         case 'src':
-          $orderByClause = " ORDER BY cc1.display_name {$dir}";
+          $orderByClause = " ORDER BY cc2.display_name {$dir}";
```

Comment:
Would it make more sense to order by 'Contact 1' still, since that's the first column in the table?
(Ideally ordered by surname, but again not in scope for this).